### ISSUE TITLE
Checksum Definition

### ISSUE THREAD

**Issue by riannella** [spdx/spdx-spec#500]
Hi, the definition for Checksum here: https://spdx.org/rdf/terms/#d4e1930
says "A Checksum is value that allows the contents of a file to be authenticated".

We think that a more appropriate word "integrity" should be used instead of "authenticated".
See: https://en.wikipedia.org/wiki/Checksum

Perhaps update to "A Checksum is a value that allows to check the integrity of the contents of a file'

**Comment by goneall** [comment]
Good catch - completely agree

**Comment by kestewart** [comment]
Setting this to an issue to be clarified in the 2.3 specification.

**Comment by paulmillar** [comment]
Hi @riannella,

#645 was just accepted to v2.2 branch.  You may like to check whether that change has also addressed the problem you described in this issue.

**Comment by riannella** [comment]
Can you post here what the updated definition for checksum is now?

**Comment by paulmillar** [comment]
Here you go:

```
### 8.4.1 Description

This field provides the result from some specific checksum algorithm.
Checksum algorithms consume data from the file and produce a short,
fix-sized summary that is sensitive to changes in the file's data.
Any random change to the file's data will (with high likelihood)
result in a different checksum value.

The metadata for the file checksum field is shown in Table 39.

[...]
### 8.4.2 Intent

Checksum values are used primarily to protect a file's data against
accidental corruption during transmissions or storage.
Cryptographically strong checksum algorithms may be used to detect
deliberate corruption, but only if the authenticity of the checksum
value may be verified.

Checksum values may also be used as an identifier for the file's data
that is (with high likelihood) unique.  Such an identifier may be used
to establish the desired version of a file, if multiple versions are
available.
```

**Comment by riannella** [comment]
Ok, that looks good. Thanks for the update

**Comment by goneall** [comment]
@riannella @paulmillar Should we close this issue as resolved with PR #645?

**Comment by kestewart** [comment]
Looks like this is resolved,  @paulmillar @riannella  - please reopen if you disagree.
